ui/packages/form/src/Dropdown.svelte

```diff
@@ -77,6 +77,7 @@
 			if (!multiselect) {
 				value = option;
 				inputValue = "";
+				showOptions = false;
```

Comment:
This just closes the dropdown after you have selected something in single mode, which is the expected behaviour (and matches the previous).

Reply:
Thanks yeah I noticed this issue as well